src/nxt_socket_msg.h

```diff
@@ -0,0 +1,65 @@
+#ifndef _NXT_SOCKET_MSG_H_INCLUDED_
+#define _NXT_SOCKET_MSG_H_INCLUDED_
+
+#if (NXT_HAVE_UCRED)
+#include <sys/un.h>
+#endif
+
+
+#if (NXT_HAVE_UCRED) && (NXT_HAVE_SOCKOPT_SO_PASSCRED)
+
+#define NXT_CRED_USECMSG    1
+#define NXT_CRED_STRUCT     ucred
```

Comment:
Why not add keyword `struct` to macro `NXT_CRED_STRUCT` to avoid latter `struct NXT_CRED_STRUCT`:
```
#define NXT_CRED_STRUCT     struct ucred
```
or declare type using `typedef`:
```
typedef struct ucred  nxt_cred_t;
```

Reply:
Will do